### PR TITLE
[#2058] Added `status_text` attribute to FeedItem, fixed issue in case_status item

### PR DESCRIPTION
**Pull request by Bartvaderkin**
Followup on #1058

### CODE REVIEW

File: src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html [hunk 1]
```diff
@@ -16,14 +16,21 @@ <h2 class="h2 {% if userfeed.action_required %}indicator{% endif %}">
...
+            <div class="userfeed-marker userfeed-marker--warning">
+                {% icon icon="warning_amber" icon_position="after" extra_classes="icon--warning" outlined=True %}
+                <span class="userfeed__status_text">{{ item.status_text }}</span>
+            </div>
+        {% else %}
+            <div class="userfeed-marker userfeed-marker--info">
+                {% icon icon="info" icon_position="after" extra_classes="icon--default" outlined=True %}
+                <span class="userfeed__status_text">{{ item.status_text }}</span>
+            </div>
+        {% endif %}
```
Comment (Bartvaderkin) [inline src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html:30]: @jiromaykin 

The status in the classnames should be dynamic, eg: `userfeed-marker--{{ item.status_indicator }}` or  `icon-{{ item.status_indicator }}`.  

So not an if/else switched on action_required, just use the value from the item.

So you have `warning`, `info`, `error` and `success` versions of the classes. The default is `info`. These are the values from `open_inwoner.openzaak.constants.StatusIndicators`

Comment (jiromaykin) [reply src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html]: This means I still needed to add conditions to retrieve the correct icons in my most recent commit, 
as I don't seem to be able to do something like this here: 
`{% include "components/StatusIndicator/StatusIndicator.html" with status_indicator=item.status_indicator .....`

**Review by jiromaykin — APPROVED** [review]
Needs to be tested with real-life data.

**Review by stevenbal — APPROVED** [review]

File: src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html (hunk 1 above)
Comment (Bartvaderkin) [reply src/open_inwoner/templates/cms/plugins/userfeed/userfeed.html]: Feels weird that doesn't work.

**Pull request merged**